### ISSUE TITLE
Inconsistency in system requirements

### ISSUE THREAD

**Issue by troncomputers** [flutter/flutter#134113]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

Macbook Air (2014)
System requirements states that `Operating Systems: macOS, version 10.14 (Mojave) or later.` but `flutter doctor` states that requires Xcode 14 or higher which I can't install on Big Sur because Xcode 13 is the highest version I am able to install.

### Actual results

Can't run any Flutter project on simulator

### Logs

<details>
<summary>Logs</summary>

```console

<!-- Paste your logs here -->

```

</details>

### Flutter Doctor output

<details>
<summary>Doctor output</summary>

```console

<!-- Paste your output here -->

```

</details>

**Comment by dam-ease** [comment]
Hi @troncomputers. Thanks for filing this. Can you provide more information on this, with Xcode 13 on Big Sur, are you able to install flutter?
Also, can you share the logs after installing flutter and running `flutter doctor -v`?

**Comment by troncomputers** [comment]
I am able to run Flutter project as MacOS application (window). I am not able to connect to iOS emulator with Visual Studio Code. On Stackoverflow someone wrote that I should check if everything is installed with `flutter doctor` command.
```
[✓] Flutter (Channel stable, 3.13.2, on macOS 11.7.9 20G1426 darwin-x64, locale pl-PL)
    • Flutter version 3.13.2 on channel stable at /Users/<user_name>/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ff5b5b5fa6 (13 days ago), 2023-08-24 08:12:28 -0500
    • Engine revision b20183e040
    • Dart version 3.1.0
    • DevTools version 2.25.0

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[!] Xcode - develop for iOS and macOS (Xcode 13.0)
    • Xcode at /Users/<user_name>/Downloads/Xcode 2.app/Contents/Developer
    • Build 13A233
    ✗ Flutter requires Xcode 14 or higher.
      Download the latest version or update via the Mac App Store.
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] VS Code (version 1.81.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.72.0

[✓] Connected device (1 available)
    • macOS (desktop) • macos • darwin-x64 • macOS 11.7.9 20G1426 darwin-x64

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 4 categories.
```

**Comment by dam-ease** [comment]
Hi @troncomputers. Thanks for explaining further. As regards this warning message from `flutter doctor` `Flutter requires Xcode 14 or higher`, this is an intentional change/update and is included in the `3.13.0` [release note](https://docs.flutter.dev/release/release-notes/release-notes-3.13.0#:~:text=Increase%20Xcode%20minimum%20version%20to%2014%20by%20%40jmagman%20in%20125639).

Secondly, about this issue you mentioned: 
> I am able to run Flutter project as MacOS application (window). I am not able to connect to iOS emulator with Visual Studio Code.

This is a tool issue which is similar to a currently open issue https://github.com/flutter/flutter/issues/104155

I'd be closing this as a duplicate of the already open issue.
Thank you!
